qvisor/src/runc/runtime/vm.rs

```diff
@@ -333,31 +333,23 @@ impl VirtualMachine {
                 MemoryDef::PHY_LOWER_ADDR + 64 * MemoryDef::ONE_MB + 2 * MemoryDef::ONE_GB;
             vms.pageTables = PageTables::New(&vms.allocator)?;
 
-            let mut opts = addr::PageOpts::Zero();
-            opts.SetPresent().SetWrite().SetGlobal();
-            #[cfg(target_arch = "aarch64")]
-            opts.SetMtNormal().SetDirty().SetAccessed();
-
+            let opts: PageOpts = PageEntryType::page_options(PageEntryType::Kernel).unwrap();
```

Comment:
Maybe `page_options` don't need to return an Option, it is ok to return a default PageOpts, add unwrap() here will add the chance to panic.

Reply:
Is thought of as a precaution, in the case of non-implemented features. Normally it should be checked if _something_ is returned and decided accordingly.
Of course, I can change it if it is deemed _"bad practice"_.